src/core/server/saved_objects/service/lib/repository.test.js

```diff
@@ -790,6 +810,34 @@ describe('SavedObjectsRepository', () => {
         });
       });
     });
+
+    it(`deserializes the raw ES response into a saved object`, async () => {
```

Comment:
I'd like to suggest that you move this test up so it's inside the `describe('returns', () => {` block.
I rewrote this test suite recently, and it was a jumbled mess -- I tried to organize the tests for each method into blocks to make it easier to review and maintain.